README.md

```diff
@@ -99,7 +99,7 @@ $ bazel run :bazel-remote-image
 ## Build a standalone Linux binary
 
 ```bash
-$ bazel build :bazel-remote
+$ bazel build --stamp --workspace_status_command=$(pwd)/git-commit.sh :bazel-remote
```

Comment:
Hmm. Consider putting this in an extra section as optional? I am somewhat worried that this will make Bazel look more complex than it is?

Reply:
Maybe I could move it into a workspace bazelrc file? Not sure if that's possible

Reply:
The .bazelrc file setup seems to work. I couldn't figure out a way to specify <the workspace path> in the file so I embedded the one-line script there instead.